dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/filter/TokenFilter.java

```diff
@@ -47,10 +47,14 @@ public Result invoke(Invoker<?> invoker, Invocation inv)
             Class<?> serviceType = invoker.getInterface();
             Map<String, Object> attachments = inv.getObjectAttachments();
             String remoteToken = (attachments == null ? null : (String) attachments.get(TOKEN_KEY));
-            if (!token.equals(remoteToken)) {
-                throw new RpcException("Invalid token! Forbid invoke remote service " + serviceType + " method " + inv.getMethodName() +
-                        "() from consumer " + RpcContext.getServiceContext().getRemoteHost() + " to provider " +
-                        RpcContext.getServiceContext().getLocalHost()+ ", consumer incorrect token is " + remoteToken);
+            if (ConfigUtils.isDefault(token)){
```

Comment:
This means that as long as the token=default is configured, the request can be executed arbitrarily, right?

Reply:
yes as described in the question